ersilia/serve/api.py

```diff
@@ -114,12 +137,66 @@ def _post(self, input, output):
             result = json.dumps(result, indent=4)
             result = self.__result_returner(result, output)
         if result is None and len(input) == 1:
-            # if the only one prediction did not work, return empty
             result = [{"input": input[0], "output": self._empty_output}]
             result = json.dumps(result, indent=4)
```

Comment:
Can we take this opportunity to take out this piece of code from the if blocks and just always do a Json dump and result returner?

Reply:
OKay @DhanshreeA.

Reply:
Hey @DhanshreeA I am getting `EmptyOutputError` after the change.

Reply:
I put it back and it works. I think `output: None` can't be processed after its get generated.